draftfast/test/data/dk-nba-pickem-salaries.csv

```diff
@@ -1,40 +1,40 @@
-Position,Name + ID,Name,ID,Roster Position,Game Info,TeamAbbrev,AvgPointsPerGame

```

Comment:
What's the reason for the change in the diff here? I just took these directly from DraftKings.

Reply:
git strangeness. i must have accidentally deleted and re-added the file in different commits, i don't think the file actually changed